pkg/drawio/iconTreeNode.go

```diff
@@ -4,6 +4,7 @@ type IconTreeNodeInterface interface {
 	TreeNodeInterface
 	RouterID() uint
 	SG() SquareTreeNodeInterface
+	setSG(SquareTreeNodeInterface)
```

Comment:
Maybe add a TODO here for supporting multiple SGs per icon

Reply:
done, also there is an issue on it